packages/table/src/utils/iconUtils.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { ReactElement, SVGAttributes } from 'react';
+import { StatusIcon } from '@synchro-charts/core';
```

Comment:
This is just a type right, is it worth it to have a dependency on synchro-charts for this?

Reply:
It is an enum as 
```
export declare enum StatusIcon {
    ERROR = "error",
    ACTIVE = "active",
    NORMAL = "normal",
    ACKNOWLEDGED = "acknowledged",
    SNOOZED = "snoozed",
    DISABLED = "disabled",
    LATCHED = "latched"
}
```
 So we would need to keep it and so it can be easier to manage constant values.